packages/openapi-v3/test/unit/controller-spec/controller-decorators/param-decorators/param.shortcut.test.ts

```diff
@@ -0,0 +1,65 @@
+import {expect} from '@loopback/testlab';
+import {post, param, getControllerSpec} from '../../../../../';
+
+describe('param shortcut', () => {
+  it('javascript primitive types', () => {
+    class MyController {
+      @post('/users')
+      createUser(
+        @param.query.string('name') name: string,
+        @param.query.number('age') age: number,
+        @param.query.boolean('isNew') isNew: boolean,
+      ) {}
+    }
+
+    const controllerSpec = getControllerSpec(MyController);
+    const params = controllerSpec.paths['/users'].post.parameters;
+    expect(params.length).to.eql(3);
+
+    expect(params[0].schema.type).to.eql('string');
+    expect(params[0].schema.format).to.eql(undefined);
+    expect(params[1].schema.type).to.eql('number');
+    expect(params[1].schema.format).to.eql(undefined);
+    expect(params[2].schema.type).to.eql('boolean');
+    expect(params[2].schema.format).to.eql(undefined);
```

Comment:
One logical assertion per test please. 

This test should be split into 3 tests - one for each type (string, number, boolean).